lib/atat-iam-stack.ts

```diff
@@ -1,7 +1,6 @@
 import * as cdk from "aws-cdk-lib";
 import * as iam from "aws-cdk-lib/aws-iam";
 import * as custom from "aws-cdk-lib/custom-resources";
-import * as statement from "cdk-iam-floyd";
```

Comment:
Unfortunate we had to un-floyd this.  So many additional lines.  But at least we can be more confident this won't break on the next CDK release.